samples/nodejs/EchoBot/README.md

```diff
@@ -0,0 +1,69 @@
+# Echo-bot
+
+This is a sample of a simple Agent that is hosted on an Node.js web service.  This Agent is configured to accept a request and echo the text of the request back to the caller.
+
+This Agent Sample is intended to introduce you to the basic operation of the Microsoft 365 Agents SDK messaging loop. It can also be used as the base for a custom Agent you choose to develop.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) version 18 or higher
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows)
+- [Bot Framework Emulator](https://github.com/Microsoft/BotFramework-Emulator/releases) for Testing Web Chat.
+
+## Running this sample
+
+**To run the sample connected to Azure Bot Service, the following additional tools are required:**
+
+- Access to an Azure Subscription with access to perform the following tasks:
+    - Create and configure Entra ID Application Identities
+    - Create and configure an [Azure Bot Service](https://aka.ms/AgentsSDK-CreateBot) for your bot
+    - Create and configure an [Azure App Service](https://learn.microsoft.com/azure/app-service/) to deploy your bot on to.
```

Comment:
Move deployment options to another section (I'd lead with ACA instead of AppService)